src/main/java/gov/cabinetoffice/gap/adminbackend/services/SpotlightBatchService.java

```diff
@@ -51,4 +85,136 @@ private SpotlightBatch getSpotlightBatch(UUID spotlightBatchId) {
                 () -> new NotFoundException("A spotlight batch with id " + spotlightBatchId + " could not be found"));
     }
 
+    public List<SpotlightBatch> getSpotlightBatchesByStatus(SpotlightBatchStatus status) {
+        return spotlightBatchRepository.findByStatus(status).orElse(new ArrayList<>());
+    }
+
+    public List<SendToSpotlightDto> generateSendToSpotlightDtosList(SpotlightBatchStatus status) {
+
+        final List<SendToSpotlightDto> sendToSpotlightDtos = new ArrayList<>();
+        final List<SpotlightBatch> spotlightBatches = getSpotlightBatchesByStatus(status);
+
+        // gav comment - think we could turn this loop into a stream to make the method
```

Comment:
Same here, is this getting actioned as part of another ticket?

Reply:
I'll remove the comment and we can refactor in future. Don't think we'll have enough time just now